### DISCUSSION TITLE
Version 1.2.0

### DISCUSSION THREAD

**Discussion by soumikshah** [soumikshah/Plutus-InvestmentTracker#45] [Announcements]
Features

- Now this application supports all the currencies. It allows you to select the currency when you install the application for the first time. Select the country name and currency will be added automatically.
- If you already have this application installed and are only using one currency, you can add one more currency from the settings tab.
- Added some animations to the application. You'll notice application changes the pages more gracefully now.
- Added back button for when you're inserting new investment detail and want to go back.
- Changed design for adding investment dialog box to show the user if the field is important or not.
- Added about us section in the settings tab. The settings tab is more cleaner now. You can connect with me using more way than email now, so would be easier to share feedback and bug reports.

Bug fixes
- Some under hood cleaning for fixing few abnormalities with the app

<hr /><em>This discussion was created from the release <a href='https://github.com/soumikshah/Plutus-InvestmentTracker/releases/tag/1.2.0'>Version 1.2.0</a>.</em>

**Comment by Moises9503** [comment]
Perfecto
